### PR TITLE
fix: filter-prod flag including all workspace pkgs

### PR DESCRIPTION
**Pull request by noorulh06**
This is the followup PR for previous PR: #4957

### CODE REVIEW

**Review by zkochan — APPROVED** [review]

File: .changeset/large-suns-relax.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"pnpm": minor
```
Comment (zkochan) [inline .changeset/large-suns-relax.md]: it is a patch change.

File: .changeset/large-suns-relax.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+fix: filter-prod flag including all workspace pkgs
```
Comment (zkochan) [inline .changeset/large-suns-relax.md]: add a better description. It will be on the release page.

File: .changeset/large-suns-relax.md (hunk 1 above)
Comment (noorulh06) [reply .changeset/large-suns-relax.md]: Ok thanks. I'll update that.

**Comment by noorulh06** [comment]
I think merge build has broken due to failing of a test case on `CI / windows-latest / Node.js 14.6 (push)`. I believe it's a flaky test failure. What can be done in this case? I see no new release package created after changes are merged.

**Comment by zkochan** [comment]
The release is triggered manually. I did not make a release yet.

**Pull request merged**